BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+
+**Matthias** - :  And I've also spoken to Kevin Hammond from IOG last Saturday actually, about again, trying to get back the researchers from IOG into the conversation. So Kevin was quite surprised actually that this didn't happen because, to quote him, "Usually researchers are quite happy to provide feedback on their work if given the opportunity." So he's going to change that a bit and see if people can actually get into this study group that we've been trying to push up last time already, yeah, and get that moving. So I've put that in his hands so far. With the Vasil out of the way now, he has a bit more availability to pursue that on the IOG side, so yeah, fingers crossed and we'll see, I guess in the next biweekly, if this has lead to anything. I should also probably mention that on the CIP PR itself so that people who are currently discussing are also aware, and then maybe up to the author to see who he wants to include in the conversation from his side as well.
+
+**Matthias** - :  Yeah. So beyond that, I haven't gone through all the comments. We could go through the comments as part of this current biweekly as well, since we have a bit of time left, or we could leave that for another time and try to do again a summary of the conversation that had been happening. Any thoughts on that?
+
+**Sebastien** -:  Yeah. For this one, I feel like the summaries are nice, but I feel this one needs more than just a summary.
+
+**Matthias** - :  Of course. And as I said last time, I think this one can only move forward. At some point with regards the participation of IOG research. Without that, I'd say it's very difficult for that proposal to make concrete steps. And yeah, we have, I think two, yeah, Christopher and Colette. I think you are here both of you probably for the CIP 50, so maybe you have all sorts of stuff you want to discuss or say, please bring them up. Yeah, so I guess same situation for everyone. So I don't really have anything to add mainly in this call. We're talking about safety process in general, okay. And okay, so both of you. So is there any other items then we could discuss today before we wrap it up on maybe CIP 58? I don't think we have talked about that one yet. It was proposed and then proposed again in a new format, I think. So it's not from the same author, but it's from the same team as far as I understood, so it's superseding the previous proposal. So the first proposal was... Where it is? I'd lost it now.
```

Comment:
It's always weird in the transcript when I am responding to the chat, but the chat isn't visible in the transcript. So it just looks like I am babbling stuff at random :shrug: